src/main/kotlin/no/nav/syfo/behandler/BehandlerService.kt

```diff
@@ -239,6 +239,22 @@ class BehandlerService(
     fun getKontor(): List<PBehandlerKontor> =
         database.getAllBehandlerKontor()
 
+    fun disableKontorIfExistsOtherValidKontorWithSameHerId(
+        behandlerKontor: PBehandlerKontor,
+        partnerIds: List<Int>,
+    ): Boolean {
+        behandlerKontor.herId?.let {
+            if (database.getBehandlerKontorByHerId(it).find { other ->
```

Comment:
Går evt an å bruke `.any` i stedet for `.find` når man bare trenger å vite om det eksisterer.